src/DynamoCoreWpf/Services/UsageReportingManager.cs

```diff
@@ -148,8 +148,6 @@ public void CheckIsFirstRun(Window ownerWindow, IBrandingResourceProvider resour
             // First run of Dynamo
             if (dynamoModel.PreferenceSettings.IsFirstRun)
             {
-                FirstRun = false;
-
```

Comment:
Try not to touch this `UsageReportingManager`, it is a very sensitive class (it has legal implications if not done well). We should probably pass the value of `PreferenceSettings.IsFirstRun` to the `StartPage` before it is changed.